src/components/Expression/OperandAutocomplete.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+  InputHTMLAttributes,
+  forwardRef,
+  ComponentProps,
+  useState,
+} from "react";
+import * as Portal from "@radix-ui/react-portal";
+import { classNames } from "~/utils";
+import { AutosizeInput, InputContainer } from "./AutosizeInput";
+import { Box } from "../Box";
+import {
+  dropdownContent as dropdownContentStyles,
+  dropdownItem as dropdownItemStyles,
+  autocompleteContainer as autocompleteContainerStyles,
+  autocompleteContent as autocompleteContentStyles,
+} from "./Expression.css";
+
+export const AutocompleteItem = forwardRef<
+  HTMLElement,
+  ComponentProps<typeof Box>
+>(({ children, ...props }, ref) => {
+  return (
+    <Box className={dropdownItemStyles()} {...props} ref={ref}>
+      {children}
+    </Box>
+  );
+});
+
+AutocompleteItem.displayName = "AutocompleteItem";
+
+type OperandAutocompleteProps = Omit<
+  InputHTMLAttributes<HTMLInputElement>,
+  "color" | "width" | "height" | "size"
+> & {
+  children: React.ReactNode;
```

Comment:
question: you have a couple of imports from react above bo you refer to ReactNode via React global, why not import this directly?